package.json

```
@@ -3,7 +3,7 @@
   "private": true,
   "version": "2.0.0",
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.15.1"
```

We should keep this to the smallest version possible. The same goes for all the 'engines' fields in all `package.json` files.
```suggestion
    "node": ">=16.0.0"
```

---

requested changes made :)